content/tutorials/build-apps/build-a-browser-wallet-in-js.md

```diff
@@ -0,0 +1,191 @@
+targets:
+  - en
+  - ja # TODO: translate this page
+blurb: Build a graphical browser wallet for the XRPL using Javascript.
+---
+# Build A Browser Wallet Using JS
+<!-- STYLE_OVERRIDE: wallet -->
+
+This tutorial demonstrates how to build a browser wallet for the XRP Ledger using the Javascript programming language and various libraries. This application can be used as a starting point for building a more complete and powerful application, as a reference point for building comparable apps, or as a learning experience to better understand how to integrate XRP Ledger functionality into a larger project.
+
+## Prerequisites
+
+To complete this tutorial, you should meet the following guidelines:
+
+1. You have [Node.js](https://nodejs.org/en/download/) v14 or higher installed.
+2. You have [Yarn](https://yarnpkg.com/en/docs/install) (v1.17.3 or higher) installed.
+3. You are somewhat familiar with coding with JavaScript and have completed the [Get Started Using JavaScript](get-started-using-javascript.html) tutorial.
+
+## Source Code
+
+You can find the complete source code for all of this tutorial's examples in the [code samples section of this website's repository]({{target.github_forkurl}}/tree/{{target.github_branch}}/content/_code-samples/build-a-wallet/js/).
+
+## Goals
+
+At the end of this tutorial, you should be able to build a simple XRP wallet displayed below.
+
+![Home Page Screenshot](img/js-wallet-home.png)
+
+This application can:
+
+- Show updates to the XRP Ledger in real-time.
+- View any XRP Ledger account's activity, including showing how much XRP was delivered by each transaction.
+- Show how much XRP is set aside for the account's [reserve requirement](reserves.html). 
+- Send [direct XRP payments](direct-xrp-payments.html), and provide feedback about the intended destination address, including:
+    - Displaying your account's available balance
+    - Verifying that the destination address is valid
+    - Validating the account has enough XRP to send
+    - Allowing you to specify a destination tag
+
+## Steps
+
+Before you begin, make sure you have the prerequisites installed. Check your node version by running `node -v`. If necessary, [download Node.js](https://nodejs.org/en/download/).
+
+**Tip:** If you get stuck while doing this tutorial, or working on another project, feel free to ask for help in the XRPL's [Developer Discord](https://discord.com/invite/KTNmhJDXqa).
+
+### 1. Set up the project
+
+1. Navigate to the directory that you want to create the project in.
+2. Run 
+   ```sh
+    yarn create vite-app <project-name>
+    ```
+    to create a new project.
+
+3. Create or modify the file `package.json` to have the following contents:
+
+{{ include_code("_code-samples/build-a-wallet/js/package.json", language="js") }}
+
+   - Alternatively you can also do `yarn add <package-name>` for each individual package to add them to your `package.json` file.
+  
+4. Run `yarn` to install the packages.
```

Comment:
I accidentally glossed over this step while following the tutorial. To make it harder to miss, let's put the command in a code block:

```suggestion
4. Install dependencies:

        yarn
```